### PR TITLE
Fix the linking to apps by abbrev package name

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/eXist-db/documentation/issues/154

@duncdrum Here you go buddy.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,6 +102,23 @@
     </reporting>
 
     <build>
+        <resources>
```
Comment (duncdrum) [inline pom.xml]: out of curiosity why are we first including and then excluding the same resource

**Comment by adamretter** [comment]
@joewiz exactly

**Comment by duncdrum** [comment]
to be more precise
```bash
Downloading from clojars.org: http://clojars.org/repo/ro/kuberam/maven/plugins/kuberam-expath-plugin/0.5.0-SNAPSHOT/kuberam-expath-plugin-0.5.0-SNAPSHOT.pom
[WARNING] The POM for ro.kuberam.maven.plugins:kuberam-expath-plugin:jar:0.5.0-SNAPSHOT is missing, no dependency information available
Downloading from clojars.org: http://clojars.org/repo/ro/kuberam/maven/plugins/kuberam-expath-plugin/0.5.0-SNAPSHOT/kuberam-expath-plugin-0.5.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.180 s
[INFO] Finished at: 2017-12-15T18:08:01+01:00
[INFO] Final Memory: 10M/192M
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin ro.kuberam.maven.plugins:kuberam-expath-plugin:0.5.0-SNAPSHOT or one of its dependencies could not be resolved: Could not find artifact ro.kuberam.maven.plugins:kuberam-expath-plugin:jar:0.5.0-SNAPSHOT in clojars.org (http://clojars.org/repo) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by adamretter** [comment]
@duncdrum That just needs @claudius108 to publish version 0.5.0 of the kuberam-expath-plugin to Maven Central.

**Comment by duncdrum** [comment]
I ll keep my eyes peeled, I ll test as soon as update to kuberam is live, and go ahead and merge afterwards.

**Comment by claudius108** [comment]
Hi, 

The version 0.5.0 of kuberam-expath-plugin is available.

Thanks for help,
Claudius

**Comment by adamretter** [comment]
@claudius108 Thanks mate, your a legend :-)

**Comment by claudius108** [comment]
Well, not yet. I just need to be as learned as you are ... :)

**Pull request merged**